src/world.rs

```diff
@@ -110,11 +110,24 @@ impl World {
         self.hosts.insert(addr, Host::new(addr));
     }
 
-    /// Send `message` from `src` to `dst`. Delivery is asynchronous and not
-    /// guaranteed.
+    /// Send `message` from `src` to `dst`.
+    /// Delivery between hosts is asynchronous and not guaranteed.
+    /// localhost communication does not use a topology.
     pub(crate) fn send_message(&mut self, src: SocketAddr, dst: SocketAddr, message: Protocol) {
```

Comment:
As discussed offline let's see how it would look to send via the normal code path in top, but only support "drop" manipulation.

Reply:
Made it happen.